docs/product-release-notes/src/main/asciidoc/rn-rhpam-701-con.adoc

```diff
@@ -0,0 +1,58 @@
+* Indexing storage can be very slow  and might time out [https://issues.jboss.org/browse/RHPAM-1083[RHPAM-1083]]
+* Some Process Designer properties are not disabled in View mode [https://issues.jboss.org/browse/RHPAM-1051[RHPAM-1051]]
+* Process Instance data cannot be loaded from Sybase 16.0 [https://issues.jboss.org/browse/RHPAM-1043[RHPAM-1043]]
+* Manually-added dependencies do not show up in the Dependencies list [https://issues.jboss.org/browse/RHPAM-1039[RHPAM-1039]]
+* Business Central and Business Central Monitoring do not apply the correct style to the navigation bar [https://issues.jboss.org/browse/RHPAM-985[RHPAM-985]]
+* The change from server "template" and "container" to "configuration" and "deployment unit" is not consistent across UI [https://issues.jboss.org/browse/RHPAM-955[RHPAM-955]]
+* Newly created persistable data objects are not automatically added to the list of Project Persistable Data Objects [https://issues.jboss.org/browse/RHPAM-938[RHPAM-938]]
+* Milestones: Tooltip with sort order hangs after sorting [https://issues.jboss.org/browse/RHPAM-912[RHPAM-912]]
+* Guided Decision Table: Cannot add a row into table with a work item [https://issues.jboss.org/browse/RHPAM-666[RHPAM-666]]
+* Tasks: First Due Date update writes null to task log [https://issues.jboss.org/browse/RHPAM-609[RHPAM-609]]
+* Switching to Project view redirects to the list of assets [https://issues.jboss.org/browse/RHPAM-556[RHPAM-556]]
+* Error appears after using the Contributors tab [https://issues.jboss.org/browse/RHPAM-535[RHPAM-535]]
+* It is possible to create two project with the same name when using Duplicate [https://issues.jboss.org/browse/RHPAM-533[RHPAM-533]]
+* In Process Designer, open an XML editor instead of a 'sorry' screen when a process cannot be opened [https://issues.jboss.org/browse/RHPAM-447[RHPAM-447]]
+* Process Designer Toolbox icons are cut-off and animations are slow [https://issues.jboss.org/browse/RHPAM-431[RHPAM-431]]
+
+== {KIE_SERVER} 
+
+* KieScanner does not load new rules [https://issues.jboss.org/browse/RHPAM-1124[RHPAM-1124]]
+* Execution Servers: ContainerNotFound error thrown when removing the container [https://issues.jboss.org/browse/RHPAM-899[RHPAM-899]]
+
+== {PRODUCT} on OpenShift
+
+* Broken link to rhpam70-businesscentral-indexing-openshift in image stream file [https://issues.jboss.org/browse/RHPAM-1163[RHPAM-1163]]
+* Maven repo parameter issues in OpenShift templates [https://issues.jboss.org/browse/RHPAM-1166[RHPAM-1166]]
+* Remove the duplicated KIE_SERVER_BYPASS_AUTH_USER required parameter from KIE server template [https://issues.jboss.org/browse/RHPAM-1152[RHPAM-1152]]
+* Drools 7 KieServices.Factory.get() returns null [https://issues.jboss.org/browse/RHPAM-1150[RHPAM-1150]]
+* AMQ readiness check fails on the rhpam70-authoring-ha OpenShift template [https://issues.jboss.org/browse/RHPAM-1150[RHPAM-1150]]
```

Comment:
Copy paste error ? The link points to different issue - based on title the link should point to be to https://issues.jboss.org/browse/RHPAM-1141